doc/main.dox

```diff
@@ -184,7 +184,7 @@ sockperf: [tid 4805] using epoll() to block on socket(s)
                                  optional: set dummy-send rate per second (default 10,000), usage: --dummy-send [<rate>|max]
  -t      --time                 -Run for <sec> seconds (default 1, max = 36000000).
          --client_port          -Force the client side to bind to a specific port (default = 0).
-         --client_ip            -Force the client side to bind to a specific ip address (default = 0).
+         --client_addr          -Force the client side to bind to a specific address [IPV4/6, UDS_NAME] (default = 0).
```

Comment:
please consider
`specific address in IPv4, IPv6, UNIX domain socket format (default = 0)`

**Renaming should be confirmed by verification!**